test/common/verify_3scale_UIBBT_alerts.go

```diff
@@ -70,15 +66,15 @@ func TestThreeScaleUIBBTAlerts(t TestingTB, ctx *TestingContext) {
 	time.Sleep(1 * time.Minute)
 
 	// Scale down system-app to 0 pods
-	err = scaleDeploymentsConfig(t, "system-app", ThreeScaleProductNamespace, 0, client)
+	err = scaleDeployments(context.TODO(), t, client, ThreeScaleProductNamespace, "system-app", 0)
 
 	defer func() {
-		err = scaleDeploymentsConfig(t, "system-app", ThreeScaleProductNamespace, 1, client)
+		err = scaleDeployments(context.TODO(), t, client, ThreeScaleProductNamespace, "system-app", 1)
```

Comment:
I think that the most common no. of replicas for system-app is 3, so let's hard-code 3 rather than 1 as a quick improvement. And make a proper fix in a different PR since it is not related to moving away from DeploymentConfig, wdyt?